Assets/Character/Scripts/DirectionChecker.cs

```diff
@@ -0,0 +1,38 @@
+﻿using UnityEngine;
```

Comment:
what kind of direction does this class return? Current - prev position may return right even though the player is facing left. How is this class different from getting the rigidbody's velocity?